DiskIOMeter.c

```diff
@@ -73,7 +74,8 @@ static void DiskIOMeter_updateValues(Meter* this) {
 
       if (data.totalBytesWritten > cached_write_total) {
          diff = data.totalBytesWritten - cached_write_total;
-         diff /= 1024; /* Meter_humanUnit() expects unit in kilo */
+         diff /= ONE_K; /* Meter_humanUnit() expects unit in kilo */
+         diff = (1000 * diff) / passedTimeInMs; /* convert to per second */
          cached_write_diff = (uint32_t)diff;
```

Comment:
What if diff >= 4 GiB?

Reply:
AFAICS `diff` is `uint64_t`, thus we'd need to be `diff > ~16EiB` for that overflow …

Reply:
`diff` is per second (on the last line), so the `uint32_t` cast would truncate it at 4 GiB/s wouldn't it?

Reply:
Actually the value is in kilo. So the truncation would happen with 4TiB/s, no?

Reply:
Ah, my bad, it's in KiB/s..

Reply:
Why no single division?

Reply:
Trying to keep the code in sync with network Io meter. Any opinions from upstream?